.github/workflows/test.yml

```diff
@@ -6,7 +6,8 @@ name: Run Tests
 on:
   push:
     branches: [master]
-  pull_request:
+  pull_request: {}
+  workflow_dispatch: {}
```

Comment:
Please don't change this. It's unrelated.

Reply:
I pushed this incorrectly -- I have reverted it.